composer.json

```diff
@@ -12,18 +12,18 @@
     "require": {
         "php": "^5.3.9|^7.0",
         "webmozart/path-util": "^2.2",
-        "webmozart/glob": "^4.0",
+        "webmozart/glob": "^4.1",
         "webmozart/assert": "^1.0"
     },
     "require-dev": {
-        "webmozart/key-value-store": "^1.0.0-beta5",
```

Comment:
I would not remove it as we will probably use stores in the ChangeStream. Or should we put it as a suggestion?